src/referencePackages/src/system.collections.immutable/8.0.0/System.Collections.Immutable.Manual.cs

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
Add a license header and link to a tracking issue for GenAPI to remove these manual defines.

Reply:
@michaelsimons it would be good to enabled the analyzer rule that validates the existence of a license header for the reference sources.

Reply:
@ellahathaway - See ^ as it relates to https://github.com/dotnet/source-build/issues/3759, a request to ensure the license header exists.